src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -803,8 +803,10 @@ static int rbd_nbd(int argc, const char *argv[])
       cmd = Connect;
     } else if (strcmp(*args.begin(), "unmap") == 0) {
       cmd = Disconnect;
+      g_ceph_context->_conf->set_val_or_die("admin_socket", "");
```

Comment:
@dillaman just from line 801, it begins to check the command type: map, unmap or list-mapped. But global_init is called in line 761, which is before 801... I cannot just move this global_init after 801 line. So I just set admin_socket to empty here.